src/OpenTelemetry/Trace/TracerProviderBuilderSdk.cs

```diff
@@ -129,55 +129,49 @@ internal TracerProviderBuilder AddProcessor(BaseProcessor<Activity> processor)
             return this;
         }
 
-        internal TracerProvider Build()
+        /// <summary>
+        /// Adds the ActivitySource of an instrumentation to the provider.
+        /// </summary>
+        /// <param name="instrumentationActivitySource">ActivitySource to add.</param>
+        /// <returns>Returns <see cref="TracerProviderBuilder"/> for chaining.</returns>
+        internal TracerProviderBuilder AddInstrumentationActivitySource(string instrumentationActivitySource)
```

Comment:
we dont need this, as there is already `AddSource` and this is adding anything new not already offered by `AddSource`

Reply:
Removed this.